stable/osg-hosted-ce/osg-hosted-ce/values.yaml

```diff
@@ -176,7 +176,12 @@ HTTPLogger:
 
 # If a specific node is desired for scheduling
 # set this to the hostname of the desired node
-NodeSelection: null
+NodeSelection:
+  Enabled: false
+  Hostname: null
```

Comment:
Do we need to make this first class?

Reply:
I'm not very familiar with first class, why would it need to be first class? As opposed to something like HTTPLogger (on line 171 of values.yaml).

Reply:
Hey @austinhk, sorry for the jargon! When I said "first class" here, I meant:

> Does the hostname selector really need its own special key-value pair in `values.yaml` when users could just specify this through the `NodeLabels` interface below?

In other words, I think you can just get rid of the `NodeSelection.Hostname` and let users set it directly but providing the hostname selector as a documented example as I suggested below

Reply:
Thank you for helping me understand, I think I fixed what you are referring to. Are there any other suggestions you have for anything else?